core/geometry/src/test/topology/RegionOps.test.ts

```diff
@@ -1011,7 +1017,89 @@ describe("RegionOps", () => {
     GeometryCoreTestIO.saveGeometry(allGeometry, "RegionOps", "constructAllXYRegionLoops");
     expect(ck.getNumErrors()).toBe(0);
   });
+
+  it("RegionBooleanMerge", () => {
+    const ck = new Checker();
+    const allGeometry: GeometryQuery[] = [];
+    let isArray = false, hasLoopEntries = false;
+    const arrayConsistsOfLoops = (a: Array<any>) => a.every((value: any) => value instanceof Loop);
+    const regions = IModelJson.Reader.parse(JSON.parse(fs.readFileSync("./src/test/data/curve/areaBoolean/unionRegionWithOverlappingLoops.imjs", "utf8")));
+    if (ck.testDefined(regions, "read regions from file")) {
+      if ((isArray = Array.isArray(regions)) && ck.testTrue(isArray, "regions is an array")) {
+        if (ck.testExactNumber(3, regions.length, "regions has 3 entries")) {
+          if ((hasLoopEntries = arrayConsistsOfLoops(regions)) && ck.testTrue(hasLoopEntries, "region entries are Loops")) {
+            const merged = RegionOps.regionBooleanXY(regions, undefined, RegionBinaryOpType.Union);
+            if (ck.testDefined(merged, "merge operation succeeded")) {
+              GeometryCoreTestIO.captureCloneGeometry(allGeometry, merged);
+              if (ck.testType(merged, UnionRegion, "merge operation results in a UnionRegion")) {
+                if (ck.testExactNumber(3, merged.children.length, "merged region has 3 children")) {
+                  if ((hasLoopEntries = arrayConsistsOfLoops(merged.children)) && ck.testTrue(hasLoopEntries, "merged region children are Loops")) {
+                      let totalMergedArea = 0;
+                      for (const loop of merged.children) {
+                        const area = RegionOps.computeXYArea(loop);
+                        if (ck.testDefined(area, "area computed for loop")) {
+                          totalMergedArea += area;
+                          GeometryCoreTestIO.captureCloneGeometry(allGeometry, loop, 0, 0, 50);
```

Comment:
Rather than stacking the 4 images on top of each other (via `dz`), isnt it easier to see if you stack them side by side (via `dx`)?

Reply:
I want to see them stacked on top of each other to xy operations.